src/components/data-exploration/CellInfo.jsx

```diff
@@ -35,23 +36,26 @@ const CellInfo = (props) => {
       }}
     >
       {cellInfo.cellName ? (
-        <div>
-          Cell id:&nbsp;
-          {cellInfo.cellName}
+        <div style={{ fontSize: '0.75rem' }}>
```

Comment:
Suggestion:  maybe add these in a .css file and put those divs in a class. Or maybe that's not that important if its not going to be used in many places?

Reply:
I dont' think it'll be used in any other place, but I'll refactor it and put it in a variable.